app/views/posts/index.html.erb

```diff
@@ -1,2 +1,34 @@
-<h1>Posts#index</h1>
-<p>Here is a list of posts for a given user</p>
+<div class="flex flex-col w-full">
+<div class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+
+  <div class="w-20 mr-2">
+  <%= image_tag 'img2.jpg', alt: 'icon'%>
+  </div>
+
+
+  <div class="flex justify-between w-full">
+    <%# <h2 class="flex text-bold text-2xl"><%= @post.author %></h2> %>
+    <%# <div>Number of Posts: <%= @post.comments_counter%> </div> %>
+  </div>
+
+
+  
+
+<ul class="w-full max-w-4xl mx-auto p-6 space-y-6">
+  <% @posts.each do |post| %>
+  
+    <li class="flex items-center justify-between flex-col w-full p-6 bg-white rounded-lg md:flex-row">
+      <a href="http://127.0.0.1:3000/users/<%= post.author_id%>/posts/<%= post.id%>" class="flex w-full">
+        <div class="flex justify-between w-full"><h2 class="flex text-bold text-2xl"><%= post.title%></h2>
+        <ul class="flex justify-between w-48">
+          <li>Comments: <%= post.comments_counter%></li>
+          <li>Likes: <%= post.likes_counter%></li>
+        </ul>
+        </div>
+      </a>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- Same here
```ruby
  def latest_comments
     Comment.where(post_id: id).order(created_at: :desc).limit(5)
  end
```
- [ ]  User's post index as displayed on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png) should use the `post.recent_comments` 🙃 to render 5 most recent comments in the `app/views/posts/index.html.erb` page
- Your view will look like:

 ```ruby
    <% post.recent_comments.each do |comment| %>
     <p><%= comment.author.name %>: <%= comment.text %></p> 
    <% end %>
 ```